core/src/com/unciv/ui/pickerscreens/ImprovementPickerScreen.kt

```diff
@@ -58,8 +61,23 @@ class ImprovementPickerScreen(val tileInfo: TileInfo, val onAccept: ()->Unit) :
 
             improvementButtonTable.add(image).size(30f).pad(10f)
 
+            // allow multiple key mappings to technologically supersede each other
+            var shortcutKey = improvement.shortcutKey
+            if (shortcutKey != null) {
+                val techLevel = ruleSet.technologies[improvement.techRequired]?.column?.columnNumber ?: -1
+                val otherImprovements = ruleSet.tileImprovements.values
```

Comment:
.asSequence(), if you're going to use multiple filters

Reply:
Ah, referring to the following lines - true, I remember thinking that - I probably tried the typo toSequence. Thus .size is out, use .any instead, then Studio will insist I fold the variable away... No it doesn't, so I renamed it.